pcsx2/VMManager.cpp

```diff
@@ -1980,26 +1981,24 @@ u32 VMManager::DeleteSaveStates(const char* game_serial, u32 game_crc, bool also
 	return deleted;
 }
 
-bool VMManager::LoadState(const char* filename)
+bool VMManager::LoadState(const char* filename, Error* error)
 {
 	if (Achievements::IsHardcoreModeActive())
 	{
-		Host::AddIconOSDMessage("LoadStateHardcoreBlocked", ICON_FA_TRIANGLE_EXCLAMATION,
```

Comment:
Is there a way for us to keep these icons here with Error::SetString or no?

Reply:
It looks like I'll need to make some further changes to how error handling works for this PR anyway. If we want to have different icons for different errors that can be worked out later I think.